app/forms/concerns/hyku_addons/work_form.rb

```diff
@@ -17,15 +17,36 @@ def editor_list
     end
 
     class_methods do
```

Comment:
This looks like a really nice clean up. The only downside is it is going to whitelist every attribute for all the work types, right?

Reply:
Yea, but i'm not sure thats an issue personally. It's better than missing some and then losing data IMO.